avc/src/planner/planner.h

```diff
@@ -53,9 +52,20 @@ class planner {
 		double theta;
 	};
 
-	pose calculateStep(double x, double y, double theta, double speed, double steer_angle, double timestep);
-	double calculatePathCost(double velocity, double steer_angle, pcl::PointCloud<pcl::PointXYZ>::Ptr Map);
-	int costAtPose(pose step, pcl::PointCloud<pcl::PointXYZ>::Ptr Map);
+	struct sim_path
+	{
+		std::vector<double> angles;
+		std::vector<pose> poses;
+		std::vector<double> speeds;
+	};
+
+	pose calculateStep(double speed, double steer_angle, double timestep);
```

Comment:
Instead of making an overload for this function, just give pStart a default value in the full signature.
ie: `pose pStart = pose{0,0,0}`